packages/wrangler/src/__tests__/helpers/msw/handlers/deployments.ts

```diff
@@ -57,3 +57,21 @@ export const mswSuccessDeployments = [
 			)
 	),
 ];
+
+export const mswSuccessLastDeployment = [
+	rest.get(
+		"*/accounts/:accountId/workers/services/:scriptName",
+		(req, res, ctx) => {
+			return res.once(
+				ctx.json({
```

Comment:
Another candidate for createFetchResult?

Reply:
Planned for https://github.com/cloudflare/wrangler2/issues/2307